clay/claycli
# #127: Update colors and icons of the page

Update style and icons of the page

Issue: https://github.com/clay/claycli/issues/127

# Header
Before:
![screen shot 2019-02-15 at 2 46 41 pm](https://user-images.githubusercontent.com/11743692/52877627-f0d20e00-3130-11e9-834c-99b443f2b167.png)

After:
![screen shot 2019-02-15 at 2 49 17 pm](https://user-images.githubusercontent.com/11743692/52877656-00515700-3131-11e9-8973-7c6e94279b97.png)

# Footer
Doesn't have logo anymore

# Links in the content
From `#2e8555` to `#88BBD6`

# Step to test
- Go to the `website` folder
- Install the dependencies using `npm install`
- Use `npm start`
- Go to [localhost](http://localhost:3000/claycli/docs/cli.html)
- Check the `header`,  `footer` and `links on the content`
